Rakefile

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "bundler/gem_tasks"
+require "rubocop/rake_task"
+
+begin
+  require "rspec/core/rake_task"
+  RSpec::Core::RakeTask.new(:spec)
+rescue LoadError
+  # Ignored
```

Comment:
Please, remove redundant comments

Reply:
This is a rubocop's comment, because the rescue block cannot be empty